File: 02. Algorithms/03. Searching/Problems/16. 167. Two Sum II - Input array is sorted/brute-force.java
// https://leetcode.com/problems/two-sum-ii-input-array-is-sorted/
class Solution {
    public int[] twoSum(int[] arr, int target) {
        int answer[] = {0 , 0};
        for(int i = 0; i < arr.length; i++) {
            for(int j = i + 1; j < arr.length; j++) {
                if((arr[i] + arr[j]) == target) {
                    answer[0] = i + 1;
                    answer[1] = j + 1;
                    break;
                }
            }
        }
        return answer;
    }
}

File: 02. Algorithms/03. Searching/Problems/16. 167. Two Sum II - Input array is sorted/brute-force.js
/**
 * Time O(n^2) | Space O(1)
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
   for(let i = 0; i < numbers.length; i++) {
    for(let j = i + 1; j < numbers.length; j++) {
        if (numbers[i] + numbers[j] === target) 
{
    return [i+1, j+1];
}
    }
   } 
    
    return "can't obtain the sum";
};
